app/src/main/java/org/simple/clinic/facility/alertchange/AlertFacilityChangeSheet.kt

```diff
@@ -91,6 +120,13 @@ class AlertFacilityChangeSheet : BottomSheetActivity() {
     finish()
   }
 
+  private fun closeSheetWithResult() {
```

Comment:
We now have a `closeSheetWithResult` and a `closeSheet(result)` and it is not clear from the name what is the difference between the two? What do you think about renaming `closeSheetWithResult` to `closeSheetWithContinuation` or something like that?

Reply:
This will be refactored when we start using the intent with continuation.
I have made this change already in the next PR.